### DISCUSSION TITLE
TextBlockFabricIcons 找不到

### DISCUSSION THREAD

**Discussion by wangjunjx8868** [HandyOrg/HandyControl#1520] [Q&A] [answered]
按照这个文档的写法https://handyorg.gitee.io/handycontrol/extend_controls/sideMenu/
  <hc:SideMenuItem Header="Summary" Command="{Binding SelectCmd}" CommandParameter="{Binding Header,RelativeSource={RelativeSource Self}}">
            <hc:SideMenuItem.Icon>
                <TextBlock Text="&#xf2cb;" Style="{StaticResource TextBlockFabricIcons}"/>
            </hc:SideMenuItem.Icon>
        </hc:SideMenuItem>

TextBlockFabricIcons 找不到，

**Comment by sukai-cheng** [comment]
@wangjunjx8868 @cuiliang 麻烦解决啊！

**Reply by NaBian** [reply, marked as answer]
这是 demo 里的字体，不在控件库中。
